app-modules/theme/src/Filament/Pages/ManageCollegeBrandingSettings.php

```diff
@@ -92,6 +98,20 @@ public function form(Form $form): Form
             ]);
     }
 
+    public function save(): void
+    {
+        // Save the settings first
+        parent::save();
+
+        // Retrieve the updated settings
+        $settings = app(CollegeBrandingSettings::class);
+
+        // Check the specific field
+        if (! $settings->dismissible && EnableBrandingBar::active()) {
```

Comment:
You should do the Feature Flag check first here.

Reply:
Done